resources_pages/block_rep.md

```diff
@@ -0,0 +1,23 @@
+# Block Representative Guidelines
+
+To assist in this, every block, a student volunteers to represent their peers, acting as a liaison between students and MDS leadership (we/our/ours). This allows us to see how the block is going, allowing us to make changes if need be. 
+
+Here are some guidelines for the "block rep".
+
+#### 1. Introduce yourself to your peers as the new block rep
+
+We recommend doing this via Slack. If there isn't a "students only" channel, please do make one. You can try creating a private channel and then tagging `@students` in there. This will hopefully prompt you to invite all students into the channel.
+
+#### 2. Collect feedback throughout the block
+
+The main responsibility is to collect feedback from your classmates. How you do this is up to you. Some block reps use online surveys, or even invite students to meet them in person during a dedicated time.
+
+Each block, we will be dedicating a time during lecture for students to discuss the block as a class. This is a great avenue for the block rep to record some key insights. 
+
+#### 3. Attend our team meeting
+
+We'll ask you to attend our team meeting, usually about 2/3 or 3/4 through the block. Typically the block rep part of the meeting takes about 20 minutes, right at the start, but it’s flexible. We’ll continue the meeting without the block rep after they're done.
```

Comment:
they-->you, for consistency? 
Maybe add that we'll continue to reflect on / discuss what we heard?

Reply:
Maybe we can also add here (or its own section) that, optionally, they can create a short informal report (usually Markdown) which we'll archive on our end.

Reply:
Adding to Section 2